apps/web-giddh/src/app/services/common.service.ts

```diff
@@ -27,15 +27,6 @@ export class CommonService {
             }));
     }
 
-    public GetCurrency(): Observable<BaseResponse<any, any>> {
```

Comment:
@prateek-giddh Please check this code removal

Reply:
@ravinderwalkover you removed this code from all the branches but somehow it is there on giddh-2.0 branch, therefore, keeping this change.